templates/documentation/permafrost.html

```diff
@@ -84,7 +88,7 @@ <h4>Geophysical Institute Permafrost Lab (GIPL) min/mean/max summaries (point qu
       <td colspan="2">CSV output is also available by appending <code>?format=csv</code> to the URL.</td>
     </tr>
   </tfoot>
-</table>
+</table> -->
```

Comment:
Note sure what the `-->` here is, but I don't know enough about HTML perhaps. Craig will probably have a better idea, but maybe just double check this isn't an editor / linting thing. Ditto with the `!--` on L64. The documentation page is rendering just great though!

Reply:
These lines should be removed as these are HTML comments: https://github.com/ua-snap/data-api/blob/b8f5a70a6de07f4babd57d25ba801c08e4e54c80/templates/documentation/permafrost.html#L64-L91